src/Dialog/Dialog.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import classNames from 'classnames';
+
+class Dialog extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        isOpen: PropTypes.bool
+    };
+
+    componentDidMount() {
+        if(!!this.props.isOpen) {
```

Comment:
no need of `!!`. The proptype already specify the value should be a boolean. If the user provide something else, he will be notified that he has to change it.